addon/utils/general.js

```diff
@@ -12,3 +12,14 @@ export function applyClass(svg, klass) {
   // now we have 2 problems...
   return svg.replace('<svg', '<svg class="'+klass+'"');
 }
+
+// add/update title to svg
+export function applyTitle(svg, title) {
+  if (!title) { return svg; }
+
+  if (svg.includes('<title>')) {
```

Comment:
String.includes() is not supported by IE11, I think it would be safer to use `svg.indexOf('<title>') !== -1` instead.

Reply:
Sure @raido feedback applied, Travis is :green_heart:, all good for merging, what do you think?